nixos/maintainers/scripts/ec2/create-amis.sh

```diff
@@ -26,12 +26,32 @@ var ${home_region:=eu-west-1}
 var ${bucket:=nixos-amis}
 var ${service_role_name:=vmimport}
 
-var ${regions:=eu-west-1 eu-west-2 eu-west-3 eu-central-1 eu-north-1
-         us-east-1 us-east-2 us-west-1 us-west-2
+# Output of the command:
+# > aws ec2 describe-regions --all-regions --query "Regions[].{Name:RegionName}" --output text
```

Comment:
```suggestion
# > aws ec2 describe-regions --all-regions --query "Regions[].{Name:RegionName}" --output text | sort
```

I don't know what determines the current order, but we'll regret (slightly) not sorting if it changes.

Reply:
Great point!

Reply:
Addressed!